documentation/rfc/journeys_ridesharing.json

```diff
@@ -0,0 +1,683 @@
+                                                "length": 1000
+                                            }
+                                        ],
+                                        "coordinates": [
+                                            [
+                                                3.077759,
+                                                44.10067
+                                            ],
+                                            [
+                                                4.78616,
+                                                43.921963
+                                            ]
+                                        ]
+                                    },
+                                    "duration": 0,
+                                    "type": "crow_fly",
+                                    "id": "section_6_1_0",
+                                    "mode": "walking"
+                                },
+                                {
+                                    "from": {
+                                        "name": "Pickup RideSharing",
+                                        "id": "start_ridesharing"
+                                    },
+                                    "to": {
+                                        "name": "Dropoff RideSharing",
+                                        "id": "end_ridesharing"
+                                    },
+                                    "departure_date_time": "20180110T115000",
+                                    "arrival_date_time": "20180110T151000",
+                                    "additional_informations": [
+                                        "date_time_estimated"
+                                    ],
+                                    "display_informations": {
+                                        "ridesharing": {
+                                            "operator": "Instant System",
+                                            "platform": "Boogi Rennes",
+                                            "driver": {
+                                                "alias": "Johnny_75",
+                                                "image": "https://dummyimage.com/128x128/C8E6C9/000.png&text=J7",
+                                                "gender": "male",
+                                                "rating": {
+                                                    "scale_min": 0,
+                                                    "scale_max": 5,
+                                                    "value": 3.5,
+                                                    "count": 18
+                                                }
+                                            },
+                                            "available_seats": 2
+                                        },
+                                        "links": [
```

Comment:
Not sure about using the `links` of sections[0].display_informations to display application deeplink.